blocks/block-edit/index.js

```diff
@@ -50,7 +52,26 @@ export class BlockEdit extends Component {
 		};
 	}
 
+	componentDidUpdate( prevProps, prevState ) {
+		const notEqual = ! isShallowEqual( prevProps, this.props );
+		console.log( 'BlockEdit did update:', notEqual, ! isShallowEqual( prevState, this.state ) );
```

Comment:
I'm assuming we should drop these debugging changes (i.e. all of the changes to this file)?

Reply:
Yes, I have a separate commit to remove from this PR with debug only.